src/test/python/ceph-disk/tests/test_ceph_disk.py

```diff
@@ -136,7 +136,16 @@ def test_list_format_dev_plain(dev):
                     'ptype': ptype,
                     'state': 'prepared',
                 }
-                out = ceph_disk.list_format_dev_plain(dev, devices)
+                dev_latest = [{
+                     'dmcrypt': {},
+                     'path': '/dev/dm_0',
+                     'whoami': '1234',
+                }]
+                with patch.multiple(
+                        ceph_disk,
+                        list_devices=lambda path: dev_latest,
```

Comment:
Yes, that could work normal.
I return a new value due to the code path. (It could use the same return we ever)
I will change it : ), thanks for reminder!!